sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/AnalyzeConversationOptions.cs

```diff
@@ -7,18 +7,10 @@
 namespace Azure.AI.Language.Conversations
 {
     /// <summary> The request body. </summary>
-    [CodeGenModel("ConversationAnalysisOptions")]
-    [CodeGenSuppress("AnalyzeConversationOptions", typeof(string))]
-    public partial class AnalyzeConversationOptions
+    public partial class ConversationAnalysisOptions
```

Comment:
The class needs to follow the method name + "Options", so this needs to remain `AnalyzeConversationOptions`.